docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -120,6 +120,12 @@ Allows setting a sort order for the result. Defaults to `start_time`, i.e. sorti
 
 `index_count`::
   Sort snapshots by the number of indices they contain and break ties by snapshot name.
+
+`shard_count`::
+Sort snapshots by the number of shards they contain and break ties by snapshot name.
+
+`failed_shards_count`::
+Sort snapshots by the number of shards that they failed to snapshot break ties by snapshot name.
```

Comment:
> that they failed to snapshot

I believe `they` is redundant here: `the number of shards that failed to snapshot` sounds more natural to me.

> failed to snapshot break ties by snapshot name

I guess `and` is missed here: `failed to snapshot and break ties by snapshot name`